inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -111,3 +113,10 @@ def is_record_relevant(obj, *args, **kwargs):
                 len(core_keywords) == 0:
             return False
     return True
+
+
+def is_experimental_paper(obj, eng):
+    """Is the record an experimental paper."""
+    categories = get_value(obj.data, "arxiv_eprints.categories", []) + \
+        get_value(obj.data, "subject_terms.term", [])
+    return "hep-ex" in categories or "Experiment-HEP" in categories
```

Comment:
Is this the correct criteria? 
E.g. 
https://inspirehep.net/search?ln=en&ln=en&p=collaboration%3Aatlas+-subject%3Ahep-ex+&of=hb&action_search=Search&sf=earliestdate&so=d&rm=&rg=25&sc=0

shows that there 3K papers from ATLAS that are not `hep-ex`.

@annetteholtkamp what do you think? For which INSPIRE/arXiv categories shall we guess the experiment? Shall we be consverative and just opt for the safe bet of `hep-ex`/`Experiment-HEP`?